### PR TITLE
Added docker-compose.yml for smoother Docker dev

### PR DESCRIPTION
**Pull request by DCRichards**
Hello! 👋 

_TL;DR: Saw your police map on Reddit, thought it was really cool, went to look at the code, found this, thought I could offer some help RE: your docker kill issue._

## Changes

* I've added a `docker-compose.yml` file for easier development than building a tagged version and running it every time (which gets boring really quickly!). Instead, now you simply do `docker-compose build` to build the image, followed by `docker-compose up` to start. The `docker-compose.yml` file automatically names your containers and handles the port and volume mapping, so when you change things locally, these are reflected in the container.
* Now, to kill your container, you can do `Ctrl+C` or `docker-compose stop` from another tab/window/tmux pane.
* `docker-compose.yml` is also a handy place to inject env vars if you need them, see the [docs on it here](https://docs.docker.com/compose/environment-variables/).

Shout if you have any questions, or feel free to amend or decline this if I'm preaching to the converted here, but thought it might be useful!

### CODE REVIEW

**Comment by jamesfiltness** [comment]
Hey. Thanks so much for this!
